test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -163,36 +186,19 @@ public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployme
 
         Assert.assertEquals("LRA participant action", 200, response.getStatus());
 
+        // The LRA transaction could have been started via lraClient but it is useful to
```

Comment:
Please can I ask why this was moved before the restart of the container? This might end up a duplicate discussion of https://github.com/jbosstm/lra/pull/27/files#r1910297173 but I am not sure